### PR TITLE
python: fix object leak in uwsgi.workers()

### PR DESCRIPTION
**Pull request by awelzel**

### CODE REVIEW

**Comment by awelzel** [comment]
@xrmx - is this something you'd pick up? Happy to adapt the testing part if you have a better idea here.

File: tests/testworkers.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Regression test for #2056 - uwsgi.workers() leaking objects.
+"""
+import uwsgi
+import gc
+
+
+def application(env, start_response):
+    gc.collect()
+    start_objs = len(gc.get_objects())
+
```
Comment (xrmx) [inline tests/testworkers.py]: What about adding here something like the following to check that we are actually returning something?

```
    workers = uwsgi.workers()
    assert workers
    for w in workers:
        assert w['apps']
```

Comment (awelzel) [reply tests/testworkers.py]: I had that at first, but then removed it to limit it to just test for the leaking part - added back now ;-)

**Comment by xrmx** [comment]
Thanks @awelzel !

**Pull request merged**